### ISSUE TITLE
Terminal output cleared with multiple tabs [BR] 

### ISSUE THREAD

**Issue by ywanHK** [bus-stop/x-terminal#135]
- [ ] I have reset Atom to defaults prior to submitting report.
- [x] I have not reset Atom to defaults prior to submitting report.
- [x] I have tried uninstalling/reinstalling the terminal.
- [ ] I have not tried uninstalling/reinstalling the terminal.
### Description

When opening multiple tabs in the same position (i.e. right dock), output of all other tabs will be cleared automatically

### Steps to reproduce

1. Open terminal in the right dock
2. Type in any command to get some output on the screen
3. Open another terminal tab in the right dock
4. Output from the first terminal is cleared

**Expected behavior:**

Output from all non-active tabs should remain unaffected when a new terminal is opened in the same location
Output from tab should not be affected when unfocused.

**Actual behavior:**

The output is cleared when a new tab is opened in the same location or when a tab is unfocused (demonstrated by the following 2 videos)
https://www.youtube.com/watch?v=Wpq_OmmHr4w
https://www.youtube.com/watch?v=3KLlx4Vk5JQ

**Reproduces how often:**

Always

### Versions

```
> atom --version
Atom    : 1.45.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0
```

```
> apm --version
apm  2.4.3
npm  6.2.0
node 10.2.1 x64
atom 1.45.0
python 3.7.1
git 2.25.0
visual studio 2015
```
OS name and version: **Windows 10 x64**
Terminal version: **9.0.3**

**Comment by UziTech** [comment]
I am not able to reproduce this. I am on Windows 10 and it does not clear the output for my console.

![x-terminal-clear](https://user-images.githubusercontent.com/97994/89136761-de9d8880-d4fa-11ea-9d65-cefac4ad29c4.gif)

**Comment by ywanHK** [comment]
That's weird, I tried with different versions of Atom (Portable Version) from 1.45.0 to 1.50.0-beta and it all turned out the same

Can you specify what version of Atom and apm are you using?

BTW, I haven't seen this issue on my machine with version 9.0.0 or earlier, I upgraded the package from 9.0.0 yesterday and ran into this.

**Comment by the-j0k3r** [comment]
>(Portable Version)

I wonder what the differences are. and if something is interfering

>I upgraded the package from 9.0.0 yesterday and ran into this.

The resizing code that was merged which was biggest change in relation to 9.x.x I dont think that would affect this. Bt who knows.

**Comment by ywanHK** [comment]
Ok I have tried package versions 
- 9.0.0
- 9.0.1
- 9.0.2 
- 9.0.3

It seems like the problem only exists in 9.0.3 which I believe is the latest version

I have tried to install atom 1.49.0 with both the _**Installer**_ and the _**Zip file**_ but the problem persists.
So I don't think it's a problem with the atom installation.

I have also tried removing python and msys2 from the path before installing the package. But after installing package the problem is still there, so I guess the building process is OK.

**Comment by ywanHK** [comment]
> ![x-terminal-clear](https://user-images.githubusercontent.com/97994/89136761-de9d8880-d4fa-11ea-9d65-cefac4ad29c4.gif)

I noticed that your CMD version is 10.1.19042.421 while mine is 10.0.18362.959

**Comment by UziTech** [comment]
It seems the issue is in PR #129 I also removed `element-resize-detector` and changed it to use `ResizeObserver`.

I'm testing it out a bit more but I can reproduce it now.

**Comment by UziTech** [comment]
This should be fixed by #136 and released shortly. Thanks for reporting it :tada:

**Comment by ywanHK** [comment]
I'm guessing that it's fixed by now?
